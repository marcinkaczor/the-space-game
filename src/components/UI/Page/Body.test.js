/* global describe, test, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'

import PageBody from './Body'

describe('PageBody', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<PageBody><div>Hello</div></PageBody>)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders with column and child', () => {
    const column = true
    const Child = () => <div>Hello from child</div>

    const wrapper = mount(
      <PageBody column={column}>
        <Child />
      </PageBody>
    )
    expect(wrapper.prop('column')).toEqual(column)
    expect(wrapper.find(Child).length).toEqual(1)
  })
})
