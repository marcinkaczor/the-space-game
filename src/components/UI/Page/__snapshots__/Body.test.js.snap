// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageBody snapshot renders 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <div>
    Hello
  </div>
</div>
`;
